modules/core/test/v2/unit/wallet.ts

```diff
@@ -9,36 +9,32 @@ import '../lib/asserts';
 import * as nock from 'nock';
 import * as _ from 'lodash';
 
-import { Wallet } from '../../../src/';
+import { CustomSigningFunction, Wallet } from '../../../src/';
 import * as common from '../../../src/common';
 
 import { TestBitGo } from '../../lib/test_bitgo';
 
 nock.disableNetConnect();
 
 describe('V2 Wallet:', function () {
-  let bitgo;
-  let wallet;
-  let bgUrl;
-  let basecoin;
+  const bitgo = new TestBitGo({ env: 'test' });
+  bitgo.initializeTestVars();
+  const basecoin = bitgo.coin('tbtc');
+  const walletData = {
+    id: '5b34252f1bf349930e34020a00000000',
+    coin: 'tbtc',
+    keys: [
+      '5b3424f91bf349930e34017500000000',
+      '5b3424f91bf349930e34017600000000',
```

Comment:
Where do these key values come from?

Reply:
they are just made up, they are not special